bolts/losses/loss.py

```diff
@@ -0,0 +1,19 @@
+from torch import Tensor, nn
+
+
+class CrossEntropy(nn.CrossEntropyLoss):
+    def forward(self, input: Tensor, target: Tensor) -> Tensor:
+        _target = target.view(-1).long()
+        return super().forward(input, _target)
+
+
+class ClfL1(nn.L1Loss):
+    def __init__(
+        self, out_dim: int, size_average=None, reduce=None, reduction: str = 'mean'
+    ) -> None:
+        super().__init__(size_average=size_average, reduce=reduce, reduction=reduction)
+        self.out_dim = out_dim
+
+    def forward(self, input: Tensor, target: Tensor) -> Tensor:
+        _target = target.view(-1, self.out_dim).float()
```

Comment:
Why is outdim needed here? Could you not just do `target = target.viewas(input)` instead? The inputs to the L1 loss have to have the same shape.